pages/migration/wsc-53/migration_wsc-53_libraries.md

```diff
@@ -12,3 +12,16 @@ No breaking changes are expected for simple uses.
 A detailed [Guzzle migration guide](https://github.com/guzzle/guzzle/blob/master/UPGRADING.md#60-to-70) can be found in the Guzzle documentation.
 
 The explicit `sink` that was recommended in the [migration guide for WSC 5.2](migration_wsc-52_libraries.html#guzzle) can now be removed, as [the Guzzle issue #2735](https://github.com/guzzle/guzzle/issues/2735) was fixed in Guzzle 7.
+
+## Emogrifier / CSS Inliner
+
+The Emogrifier library was updated from version 2.2 to 5.0.
+Version 3.0 deprecated the `Emogrifier` class in favor of the `CssInliner` class that was added in 2.1.
+The `Emogrifier` class was subsequently removed in version 4.0.
+
+No compatibility layer was added for the `Emogrifier` class, as the Emogrifier library's purpose was for use within the email subsystem of WoltLab Suite.
```

Comment:
```suggestion
No compatibility layer was added for the `Emogrifier` class, as the Emogrifier library's (only?) purpose was to be used within the email subsystem of WoltLab Suite.
```

`(only?)` -> may-be add `only`?

Reply:
```suggestion
No compatibility layer was added for the `Emogrifier` class, as the Emogrifier library's purpose was to be used within the email subsystem of WoltLab Suite.
```
We did not communicate that it was the “only” purpose, but it certainly was not meant to be used standalone without exactly knowing what you are doing. I'll leave out the “only”.